ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/clientlibs/js/checkoutform.js

```diff
@@ -272,17 +322,22 @@ function onSaveNDeliver() {
 	if (!doValidation && checkoutcartId) {
 		$('.empty-cartid').text("");
 		getUserDeatils['cartId'] = checkoutcartId;
-		getUserDeatils['region'] = "MH";
-		getUserDeatils['region_id'] = 0;
-		getUserDeatils['country_id'] = 'IN';
-		getUserDeatils['region_code'] = 'MH';
+
+        console.log("getUserDeatils", getUserDeatils);
```

Comment:
@nitheshkumar-hcl : pl remove console logs

Reply:
Committed the changes as per the review comments